### PR TITLE
PMAPI-1829: add buckslips docs

### PR DESCRIPTION
**Pull request by julielinter**
Implements changes for [PMAPI-1829](https://lobsters.atlassian.net/browse/PMAPI-1828)

## Checklist

- [x] Up to date with `main`
- [x] All the tests are passing
  - [x] Delete all resources created in tests
- [x] Prettier
- [x] Spectral Lint
- [x] `npm run bundle` outputs nothing suspect
- [x] `npm run postman` outputs nothing suspect

## Changes
- Adds Buckslip and Buckslip Orders sections under Special Features
- Adds all relevant endpoints to new sections
- Adds Buckslips to Creatives and Letters (response only)

### CODE REVIEW

File: lob-api-public.yml [hunk 1]
```diff
@@ -2514,6 +2528,15 @@ paths:
...
 
+  /buckslips:
+    $ref: resources/buckslips/buckslips.yml
+
+  /buckslips/{buckslip_id}:
+    $ref: resources/buckslips/buckslip.yml
+
+  /buckslips/{buckslip_id}/orders:
+    $ref: resources/buckslips/buckslip_orders/buckslip_orders.yml
+
```
Comment (jho44) [inline lob-api-public.yml:2539]: If you could just move these above the `/bulk/us_verifications` endpoint to maintain the alphabetical ordering, that'd be great ^^

Comment (jho44) [reply lob-api-public.yml:2539]: LTGM otherwise!

**Review by jho44 — CHANGES_REQUESTED** [review]

**Comment by maelingmurphy** [comment]
Just viewed the docs locally using `npm run redoc` and the changes look great, Julie! There have been some inventory-related updates to the Buckslip & Buckslip Order serialized responses ([from this PR](https://github.com/lob/lob-api/pull/8477)) but we can a file a followup ticket for those updates as to not block these initial docs from going out.

**Review by jho44 — APPROVED** [review]
LGTM outside of that thing we talked about with `cards` and `buckslips` not appearing for certified or registered letters. Will also note that you added buckslips to _creatives_, not _campaigns_ (though I know they're part of the same API group...bygones).

**Pull request merged**